src/main/webapp/WEB-INF/view/jsp/personPage.jsp

```diff
@@ -143,14 +143,14 @@
 									<div class="form-group row">
 										<label for="datetimepicker-sickness-start" class="col-sm-2 col-form-label">Start date</label>
 										<div class="col-sm-10">
-											<input id="datetimepicker-sickness-start" class="appdatetimepicker dtp-start" type='text' class="form-control"
+											<input id="datetimepicker-sickness-start" class="appdatepicker" type='text' class="form-control"
```

Comment:
Do we get here default current date displayed?

Reply:
Ok. I've added new method in personPage.js and now there is a default value of current date in the sickness history modal.